Engine/gfx/ali3dsw.cpp

```diff
@@ -376,6 +376,10 @@ bool ALSoftwareGraphicsDriver::Init(int virtualWidth, int virtualHeight, int rea
 
     virtualScreen = BitmapHelper::GetScreenBitmap();
 
+    // [ER] 2014-03-13
+    // Hide the system cursor via allegro
+    show_os_cursor(MOUSE_CURSOR_NONE);
```

Comment:
I need to clarify, is it elaborate decision to put this into ALSoftwareGraphicsDriver initialization?
I mean, is this specific to using this renderer?